src/sdcv.cpp

```diff
@@ -141,13 +141,13 @@ int main(int argc, char *argv[]) try {
         data_dir = get_impl(opt_data_dir);
     }
 
-    const char *homedir = g_getenv("HOME");
```

Comment:
According to documentation:
```
Before version 2.36 this function would ignore the HOME
```
At now we support glib >= 2.6.1, may be time to move to 2.36,
but this require analyze of how old Linux distro that have 2.35 and less.

Reply:
I tought that checking HOME was a left over from old code since at https://github.com/Dushistov/sdcv/blob/357ca4d453a2e54da731fdae4bc50067b87761c8/src/sdcv.cpp#L200 it already called `g_get_home_dir` directly, so it shouldn't have worked even before with glib < 2.36. But I'll restore the check if you prefer.